platform/security/servlet/security-servlet-logout/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -37,20 +37,24 @@
 
     <reference id="securityManager" interface="ddf.security.service.SecurityManager"/>
 
-    <service interface="javax.servlet.Servlet">
+
+    <bean id="locallogout" class="org.codice.ddf.security.servlet.logout.LocalLogoutServlet">
```

Comment:
Something worth noting: You can no longer create the bean in the service tags. Simply does not work with the latest upgrades.